DXCC Stats still including deleted entities - AwardsDialog.cpp

aa5sh opened the pull request:
I noticed the when looking at code for the email earlier that the DXCC counts were still including Deleted Entities.  Mine was showing 323 instead of 320 confirmed.  I made an adjustment to correct but not sure if it is the ideal way.

aa5sh committed:
Update AwardsDialog.cpp

I noticed the when looking at code for the email earlier that the DXCC counts were still including Deleted Entities.  Mine was showing 323 instead of 320 confirmed.  I made an adjustment to correct but not sure if it is the ideal way.

foldynl commented:
I'm a little confused now. So should deleted entities be included or not? I fixed it to include deleted entities. This doesn't work?

aa5sh commented:
I would think they shouldn’t count in the summary at the top if they are
not in the detailed list by country.  They were and I put in a pull request
i thought to exclude them.  The summary at the top were counting them and
the details below were not so it was inconsistent.  I would think they
should match.  It is a little confusing now.

On Wed, Mar 19, 2025 at 5:09 AM Ladislav ***@***.***> wrote:

> I'm a little confused now. So should deleted entities be included or not?
> I fixed it to include deleted entities. This doesn't work?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/foldynl/QLog/pull/620#issuecomment-2736026441>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUEEMXQHGAT7EEYLKIOQ44L2VE64JAVCNFSM6AAAAABZJH4HPKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMZWGAZDMNBUGE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> [image: foldynl]*foldynl* left a comment (foldynl/QLog#620)
> <https://github.com/foldynl/QLog/pull/620#issuecomment-2736026441>
>
> I'm a little confused now. So should deleted entities be included or not?
> I fixed it to include deleted entities. This doesn't work?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/foldynl/QLog/pull/620#issuecomment-2736026441>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUEEMXQHGAT7EEYLKIOQ44L2VE64JAVCNFSM6AAAAABZJH4HPKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMZWGAZDMNBUGE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

foldynl commented:
>>>Would you consider having an option at the top to include/exclude deleted countries?  I just saw the count and knew it was high.

That's a good question. I'll have to look into the DXCC rules. But if DXCC accepts removed entities, then we should NOT implement such an option.

iu0dhv commented:
Hello I think I was the one to suggest to include deleted entities (also because if you want to add past QSOs it is good to have the possibility to include them with the proper dxcc attribution). I also like the possibility to see them in Award count, but I also like the idea to have a flag that allows to include or exclude them from the count of Worked/Confirmed countries on the top lines of the Award window.

foldynl commented:
I think we already solved it in the previous version.
